system/jlib/jfile.hpp

```diff
@@ -37,7 +37,7 @@ class MemoryBuffer;
 class Semaphore;
 
 enum IFOmode { IFOcreate, IFOread, IFOwrite, IFOreadwrite, IFOcreaterw };    // modes for open
-enum IFSHmode { IFSHnone, IFSHread=0x8, IFSHfull=0x10};   // sharing modes
+enum IFSHmode { IFSHnone, IFSHset=0x1, IFSHread=0x8, IFSHfull=0x10};   // sharing modes
```

Comment:
any idea why these flags used to start at 0x8?
I was worried something else was using the lower 3 bites of the flags field,
but I can't see anything else that uses the first two bytes of flags apart from the sharing mode flags.

There should be comment on 'flags' making it clear how it's proportioned.

Reply:
No idea why, it has been that way as long as I have known.  I will add a comment.

Reply:
weird, just for the record, in case ever look back, there's this in CRemoteFile::openShared
    assertex(((unsigned)shmode&0xffffffc7)==0);

Suggests 0x8, 0x10 and 0x20 were reserved for share mode.
But I still don't know why those bits were used.